src/core_modules/capture-core/utils/cachedDataHooks/useProgramFromIndexedDB.js

```diff
@@ -1,30 +1,20 @@
 // @flow
```

Comment:
This would not be part of the final PR - just a way to show how these hooks would be used.